### PR TITLE
test: add failing test for empty group by

### PR DESCRIPTION
**Pull request by sagikazarmark**
## Overview

Empty group bys lead to failing queries.

### CODE REVIEW

**Review by tothandras — APPROVED** [review]

File: internal/streaming/clickhouse_connector/query.go [hunk 1]
```diff
@@ -123,6 +123,11 @@ func (d createMeterView) toSQL() (string, []interface{}, error) {
 	orderBy := []string{"windowstart", "windowend", "subject"}
 	sortedGroupBy := sortedKeys(d.GroupBy)
 	for _, k := range sortedGroupBy {
+		// Skip empty keys
+		if k == "" {
+			continue
```
Comment (hekike) [inline internal/streaming/clickhouse_connector/query.go:128]: I would return error. I think from operational perspective it's better to be strict and surface issues.

**Comment by GAlexIHU** [comment]
Closing due to inactivity
